build.gradle

```diff
@@ -65,7 +65,10 @@ dependencies {
         findbugsAnnotations = '3.0.1u2'
 
```

Comment:
Suggestion, not mandatory...  Since you are using all the latest versions, and unless you have a reason for wanting more granular control over individual library versions, you might consider using the new-ish gradle platform that was added to CRL, so that you can just track a single version. 

Here's an example of how it was applied in buffer:

https://github.com/DataBiosphere/terra-resource-buffer/pull/161/commits/29ffa100198871862ebe5b0f570853da5fa7fd90

Current version of `bio.terra.cloud-resource-lib:platform` is still `0.2.0`

Reply:
Great suggestion thanks! Changed to use the platform instead of individual library versions. This is much better than me looking through Artifactory to figure out what the latest version of each library was.